examples/basic/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3'
+
+x-mautic-volumes:
+  &mautic-volumes
+  - ./mautic/config:/var/www/html/config:z
+  - ./mautic/logs:/var/www/html/var/logs:z
+  - ./mautic/media/files:/var/www/html/docroot/media/files:z
+  - ./mautic/media/images:/var/www/html/docroot/media/images:z
+  - ./cron:/opt/mautic/cron:z
+
+services:
+  db:
+    image: mysql:8.0
+    environment:
+      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
+      - MYSQL_DATABASE=${MYSQL_DATABASE}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+    volumes: 
+      - mysql-data:/var/lib/mysql
+    healthcheck:
+      test: mysqladmin --user=$$MYSQL_USER --password=$$MYSQL_PASSWORD ping
+      start_period: 5s
+      interval: 5s
+      timeout: 5s
+      retries: 10
+
+  mautic_web:
+    image: ghcr.io/mollux/docker-mautic:5.x-dev-apache
+    links:
+      - db:mysql
+    ports:
+      - 8001:80
+    volumes: *mautic-volumes
+
+    environment:
+      - MAUTIC_LOAD_TEST_DATA=true
+      - MYSQL_HOST=${MYSQL_HOST}
+      - MYSQL_PORT=${MYSQL_PORT}
+      - MYSQL_DATABASE=${MYSQL_DATABASE}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+    healthcheck:
+      test: curl http://localhost
+      start_period: 5s
+      interval: 5s
+      timeout: 5s
+      retries: 100
+    depends_on:
+      db:
+        condition: service_healthy
+
+  mautic_cron:
+    image: ghcr.io/mollux/docker-mautic:5.x-dev-apache
+    links:
+      - db:mysql
+    volumes: *mautic-volumes
+    environment:
+      - MAUTIC_ROLE=mautic_cron
```

Comment:
Doesn't cron and worker container also need MYSQL env variables? Or is that covered somehow?

Reply:
@ilkkao they are not needed, as they (should) purely execute console commands from Mautic itself, which has all the needed context.

Those are examples, so if you need thos for a non-standard script, you can always add those on your own docker-compose or similar setup